### PR TITLE
Remove get container info function for security reasons

### PR DESCRIPTION
**Pull request by nithu0115**
When running log collector script, we inspect docker containers by running (docker inspect <containerID> and collect information.  For security reasons, I am removing get_container_info function which had above command for all containers.. 

Thanks,
Nithish

### CODE REVIEW

File: eks-log-collector.sh [hunk 1]
```diff
@@ -363,8 +363,13 @@ get_ipamd_info() {
 
 get_sysctls_info() {
   try "collect sysctls information"
-  
+  # dump all sysctls
   sysctl --all >> "${COLLECT_DIR}"/sysctls/sysctl_all.txt 2>/dev/null
+  
+  # dump relevant sysctls
```
Comment (JasonSwindle) [inline eks-log-collector.sh]: Does not the `sysctl --all` do the same thing?  Also "for f" should be more descriptive.

**Review by JasonSwindle — CHANGES_REQUESTED** [review]
Not bad, see comment supplied.

File: eks-log-collector.sh (hunk 1 above)
Comment (nithu0115) [reply eks-log-collector.sh]: You are correct. `sysctl --all` does the same. Removing as this redundant.

**Review by JasonSwindle — APPROVED** [review]
LGTM, Ship it

**Pull request merged**